charts/base-cluster/values.schema.json

```diff
@@ -1263,6 +1263,102 @@
       },
       "additionalProperties": false
     },
+    "weaveworks": {
+      "type": "object",
+      "description": "Values for the gitops-chart",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "adminUser": {
+          "type": "object",
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "username": {
+              "type": "string"
+            },
+            "createSecret": {
+              "type": "boolean"
+            },
+            "createClusterRole": {
+              "type": "boolean"
+            },
+            "passwordHash": {
+              "type": "string"
+            }
+          }
+        },
+        "ingress": {
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+            "host": {
+              "type": "string"
+            },
+            "tls": {
+              "type": "array"
+            },
+            "className": {
+              "type": "string"
+            }
+          }
+        },
+        "oidcSecret": {
+          "type": "object",
+          "properties": {
+            "additionalKeys": {
+              "type": "object"
+            },
+            "create": {
+              "type": "boolean"
+            }
+          }
+        },
+        "replicaCount": {
+          "type": "integer"
+        },
+        "rbac": {
+          "additionalRules": {
+            "type": "array"
+          },
+          "create": {
+            "type": "boolean"
+          },
+          "impersonationResourceNames": {
+            "type": "array"
+          },
+          "impersonationResources": {
+            "type": "array"
+          },
+          "viewSecretsResourceNames": {
+            "type": "array"
+          }
+        },
+        "metrics": {
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+            "service": {
+              "type": "object",
+              "properties": {
+                "annotations": {
+                  "type": "object"
+                },
+                "port": {
+                  "type": "integer"
+                }
+              }
+            }
+          }
+        }
+      }
```

Comment:
Should the user be able to turn this off?

I don't think so

Reply:
if the user is not interessted in metrics from the dashboard he can turn it off. However, we can drop the service section because this isn"t necessary in most cases

Reply:
>if the user is not interessted in metrics from the dashboard he can turn it off

Yeah, but will that ever happen? 🤷